rust/ql/lib/codeql/rust/frameworks/stdlib/Clone.qll

```diff
@@ -15,7 +15,12 @@ final class CloneCallable extends SummarizedCallable::Range {
     )
   }
 
-  final override predicate propagatesFlow(string input, string output, boolean preservesValue) {
-    input = "Argument[self]" and output = "ReturnValue" and preservesValue = true
+  final override predicate propagatesFlow(
+    string input, string output, boolean preservesValue, string model
+  ) {
+    input = "Argument[self]" and
+    output = "ReturnValue" and
+    preservesValue = true and
+    model = "generated"
```

Comment:
Previously this had a model of `""` and it seemed to be disabled/overwritten by the generated models. The generated models include a model for `clone` on `i64`, which caused the test for this method to fail. Changing the model to `generated` or `manual` fixed the problem. I just went with `generated` without worrying too much as this is temporary anyway.